src/test.ml

```diff
@@ -240,6 +240,33 @@ let test_update_join = [
   ];
 ]
 
+let test_param_not_null_by_default = [
+  tt "CREATE TABLE test15 (a INT, b INT NULL, c TEXT NULL)" [] [];
+  tt "CREATE TABLE test16 (d INT)" [] [];
+  tt {| 
+    SELECT a FROM test15 
+    WHERE a = @a 
+    AND a + b = @ab 
```

Comment:
what about `AND a + @x = 10`?

Reply:
It's related to the previous comment, so let's keep it for the following PR as it decided